lib/charms/mysql/v0/s3_helpers.py

```diff
@@ -74,7 +74,7 @@ def upload_content_to_s3(content: str, content_path: str, s3_parameters: Dict) -
 
 def _compile_backups_from_file_ids(
     metadata_ids: List[str], md5_ids: List[str], log_ids: List[str]
-) -> Tuple[str, str]:
```

Comment:
Libpatch needs to be updated

Reply:
fixed in https://github.com/canonical/mysql-operator/pull/204/commits/6fd06a2660180876f4a968fe91c8e58213092650